homeassistant/components/modbus/binary_sensor.py

```diff
@@ -57,10 +57,15 @@ async def async_update(self, now=None):
         )
         self._call_active = False
         if result is None:
+            if self._lazy_errors:
+                self._lazy_errors = self._lazy_errors - 1
```

Comment:
We can adjust all of these to subtract and assign in one go.

Reply:
Of course, done.